source/lac/full_matrix.cc

```diff
@@ -23,11 +23,10 @@ DEAL_II_NAMESPACE_OPEN
 
 #ifndef DOXYGEN
 
-#  ifndef DEAL_II_WITH_COMPLEX_VALUES
-// instantiate for std::complex<double> because we use it internally in
-// FESeries.
+// We use FullMatrix<std::complex<T>> for complex eigenvalues so unconditionally
+// instantiate it:
+template class FullMatrix<std::complex<float>>;
 template class FullMatrix<std::complex<double>>;
```

Comment:
Oh, we unconditionally instantiate it for `complex<double>` but not `complex<float>`? That is funny!